pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for distcc distributed compiler.
+> More information: <https://distcc.github.io/>.
+
+- Start a daemon with default settings:
```

Comment:
Can you run multiple daemons? If not, I think `Start the daemon ...` would be more fitting.

Reply:
Technically it is possible to start a few daemons, each listening to a different port. No idea why would anyone want that, but I wouldn't imply it is not possible.